makeitdark.py

```diff
@@ -93,9 +103,32 @@
     print("Found {0}".format(most_recent))
     slack_theme_path = os.path.join(slack_root_path, most_recent, "resources", "app.asar.unpacked", "src", "static", "ssb-interop.js")
 
-f = open(slack_theme_path, "a+")
-f.write("\n" + injectable)
-f.close()
-print("Your slack theme has been updated, please restart slack")
-exit()
+if undo_mode:
+    with open(slack_theme_path, "r+") as f:
+        s = ""
+        if "/* BEGIN makeitdark */" not in f.read():
```

Comment:
This works, however I think we can avoid this logic by leaving our traversal of the file up to the later for-loop.
The existing logic in the for loop can stay but if we break out of it without exiting then we know the caller hasn't made it dark.